inngest-spring-boot-adapter/src/main/java/com/inngest/springboot/InngestController.java

```diff
@@ -33,10 +34,14 @@ public ResponseEntity<String> put() {
 
     @PostMapping()
     public ResponseEntity<String> handleRequest(
+        @RequestHeader(name = "X-Inngest-Signature", required = false) String signature,
+        @RequestHeader(name = "X-Inngest-Server-Kind", required = false) String serverKind,
         @RequestParam(name = "fnId") String functionId,
         @RequestBody String body
     ) {
         try {
+            SignatureVerificationKt.checkHeadersAndValidateSignature(signature, body, serverKind, commHandler.getConfig());
+
```

Comment:
From reading https://github.com/inngest/inngest/blob/main/docs/SDK_SPEC.md#413-requirements-when-receiving-requests, I would have expected to do signature validation on GET and PUT as well, but it looks like the other SDKs only do it in POST.

inngest-js: https://github.com/inngest/inngest-js/blob/097ecc8e871f68f9e79efe89d6437576ad705388/packages/inngest/src/components/InngestCommHandler.ts#L702-L710
inngestgo: https://github.com/inngest/inngestgo/blob/5081aafecb1c7894ee70d35b7a2347a0c79887f2/handler.go#L399-L429

Reply:
yeah, it's only POST atm.
PUT basically ask the SDK to register via a separate HTTP call, and get just returns a response saying the setup is correct or not, so there are no needs.